File: 1º Semestre/Programação de Computadores/Dev C++ exercícios/Lista de exercício 03/8 - while.cpp
// Elabore um algoritmo que imprima a soma dos 100 primeiros números inteiros positivos. (while)

#include <stdio.h>

int main(){
	int numero = 1;
	int soma = 0;
	
	while(numero<=100){
		soma = soma + numero;
		numero = numero + 1;
	}
	
	printf("A soma e %4d.", soma);
}

File: 1º Semestre/Programação Python/Listas de Exercícios/Exercício 3/8 - while.py
# Elabore um algoritmo que imprima a soma dos 100 primeiros números inteiros positivos.

numero = 1
soma = 0

while numero <= 100:
    soma = soma + numero
    numero += 1

print(f"A soma dos primeiros 100 números inteiros positivos é de {soma}.")
